### PR TITLE
Remove clickEvents from subs landing page

### PR DESCRIPTION
**Pull request by JustinPinner**
## Why are you doing this?
We've shut down the tests, so all traffic now passes through the new landing pages. This PR eliminates the need for `clickEvents` used to record test participation.

[**Trello Card**](https://trello.com/c/yaceoeDS/1887-switching-subs-landing-pages-live)

## Screenshots
N/A

### CODE REVIEW

**Comment by JustinPinner** [comment]
Hi @rupertbates, that thought occurred to me late last night and I was just about to add exactly that comment myself. I'll do that and update the PR shortly.

File: assets/components/digitalSubscriptions/digitalSubscriptions.jsx [hunk 1]
```diff
@@ -45,7 +37,7 @@ const gridImageProperties = {
   imgType: 'png',
 };
 
-const appReferrer = 'utm_source=support.theguardian.com&utm_medium=subscribe_landing_page&utm_campaign=split_subscriptions_test';
+const appReferrer = 'utm_source=support.theguardian.com&utm_medium=subscribe_landing_page';
```
Comment (rupertbates) [inline assets/components/digitalSubscriptions/digitalSubscriptions.jsx]: 👍

File: assets/pages/subscriptions-landing/subscriptionsLanding.jsx [hunk 2]
```diff
@@ -19,7 +19,6 @@ import PaperSubscriptionsContainer from 'components/paperSubscriptions/paperSubs
 import { init as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
 
-import { sendTrackingEventsOnClick } from 'helpers/subscriptions';
```
Comment (rupertbates) [inline assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: I think this method might be unused in the project now in which case we could remove it as well

Comment (JustinPinner) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: You're right, `yarn validate` spotted that too 😃Should have gone now.

Comment (Ap0c) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: @rupertbates did you mean delete the implementation from `subscriptions.js` as well?

Comment (JustinPinner) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: If it's not used I should probably bin it from there too

Comment (JustinPinner) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: We can pull it back if needed later

Comment (JustinPinner) [reply assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: Having said that, I imagine we will almost definitely do more of this kind of test tracking in the near future? Maybe it's better to leave it in?
#techdebt

**Pull request merged**